### DISCUSSION TITLE
Page build failure when I change from Jekyll to Hugo

### DISCUSSION THREAD

**Discussion by IhnwhiHeo** [community/community#22233] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,<br>
I am trying to build a website on Github. I originally had built a Jekyll website. However, I removed it (the whole repository) since I wanted to create a completely new website. Then, I keep having the error messages below:</p>
<pre><code class="lang-auto">The page build failed for the `master` branch with the following error:

The variable `{{2\left( {x + 4}` on line 50 in `exampleSite/content/slides/example/index.md` was not properly closed with `}}`. For more information, see https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#tag-not-properly-terminated.

For information on troubleshooting Jekyll see:

  https://docs.github.com/articles/troubleshooting-jekyll-builds

If you have any questions you can submit a request on the Contact GitHub page at https://support.github.com/contact?repo_id=315911396&amp;page_build_id=217375615
</code></pre>
<p>I referred to websites, but I have no ideas on how to solve them. Could anyone help me, please?</p>

**Comment by tcbyrd** [comment, marked as answer]
<p>Hey there! Basically you’ll need to tell Pages not to use Jekyll. By default, Pages always attempts to parse/build files with Jekyll, but if you have files that use syntax that Jekyll doesn’t understand you can get errors like this. To tell Pages to completely skip the Jekyll build, you just need to add a <code>.nojekyll</code> file to the root of your pages branch. See this doc for more info: <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#static-site-generators" rel="noopener nofollow ugc">https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#static-site-generators</a></p>

**Comment by IhnwhiHeo** [comment]
<p>Hey! Thanks a lot for your answer, it indeed helps!<br>
Then, is it right that I add a <code>.nojekyll</code> in the <code>myname.github.io</code> repository?</p>

**Comment by tcbyrd** [comment]
<blockquote>
<p>add a <code>.nojekyll</code> in the <code>myname.github.io</code> repository?</p>
</blockquote>
<p>If that’s the repository that’s publishing your Page, yes. It needs to be in the branch that’s defined as the <code>source</code> of your Pages site. If there’s a <code>.nojekyll</code> file in the source branch, then the files in that branch are effectively what gets deployed to the back-end that serves the site.</p>

**Comment by IhnwhiHeo** [comment]
<p>Aha, I see!<br>
I appreciate your clarification, <a class="mention" href="/tcbyrd">@tcbyrd</a>!!!</p>
